param/parameterized.py

```diff
@@ -3064,8 +3058,6 @@ def __init__(mcs, name, bases, dict_):
         parameters = [(n, o) for (n, o) in dict_.items()
                       if isinstance(o, Parameter)]
 
-        mcs._param__parameters._parameters = dict(parameters)
```

Comment:
Can yo explain why this was needed and is no longer needed?

Reply:
Sure I'll try. It was only used here https://github.com/holoviz/param/pull/798/files#diff-c8fc4fb68012b66fbf42ad9e0b4714fd96a3a72b83558bc543130d78bd622eb3L1847 and as far as I can see it was just used for performance reasons (added in https://github.com/holoviz/param/pull/424) to avoid looping through the whole class `__dict__` and just through the class Parameters.